drivers/Makefile.include

```diff
@@ -2,7 +2,7 @@ ifneq (,$(filter dht,$(USEMODULE)))
     USEMODULE_INCLUDES += $(RIOTBASE)/drivers/dht/include
 endif
 ifneq (,$(filter at86rf2xx,$(USEMODULE)))
-    USEMODULE_INCLUDES += $(RIOTBASE)/drivers/at86rf2xx/include
+    USEMODULE_INCLUDES += $(RIOTBASE)/drivers/at86rf2xx/inclu
```

Comment:
My bad. Must have parked my cursor there and then accidentally pressen backspace.